### PR TITLE
JSC#DOCTEAM-562-replaced redundant link

### PR DESCRIPTION
**Pull request by Amrita42**
### PR creator: Description

Describe the overall goals of this pull request.

Replace a redundant link with a relevant one.

### PR creator: Are there any relevant issues/feature requests?

[Jira](https://jira.suse.com/browse/DOCTEAM-562)

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [ x] SLE 15 SP4/openSUSE Leap 15.4
  - [x ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by cwickert** [comment]
Before I look into the technical details, please do the following:

* Do a `git revert --hard cbb184028efd9736eb8bdb391dfd6a91e55dcc9d` to get rid of the two irrelevant commits.
* Use `git commit --ammend` to change the commit message and include the Bugzilla bug (BSC#1197535) or Jira ticket (JSC#DOCTEAM-562) numbers, otherwise in a year from now, nobody will know what this commit was about. I usually prefer Bugzilla since it is public.
* When you are done, run `git push --force` . As long as you are working in your own feature branch, `--force` is fine. Just don't do this on `main`  or `maintenance/*`, it will fail since these branches are protected.

**Comment by Amrita42** [comment]
Sure , will do

**Comment by Amrita42** [comment]
Its `git reset --hard`.  Thanks

**Comment by Amrita42** [comment]
@cwickert ptal , is there anything else required or can it be merged?

File: xml/security_ldap_active_directory_sync.xml [hunk 1]
```diff
@@ -77,7 +77,7 @@
     permission is required. For example, you have created a group named
     "Directory Server Sync". Follow the steps in the "How to grant the
     'Replicating Directory Changes' permission for the Microsoft Metadirectory Services ADMA service account"
```
Comment (cwickert) [inline xml/security_ldap_active_directory_sync.xml:79]: If you change the link, you also have to change the document title here.

**Review by cwickert — CHANGES_REQUESTED** [review]
Besides shortening the link, I don't understand what you are trying to achieve with this change.

The bug report originally was about a broken link. This was already fixed in a06809f9a8d9785f0418179ecde2c4180a789743. It was just a line break in the link, the link target did not change. 

Carla wrote this chapter in December 2021 (see c126522de29cc517a144302da77f69a059cd8125). I don't think she would have added the link if it was outdated. According to https://github.com/MicrosoftDocs/SupportArticles-docs/blob/main/support/windows-server/windows-security/grant-replicating-directory-changes-permission-adma-service.md the MS document in question is still regularly updated. Last year, it [was referenced in MS' forum](https://learn.microsoft.com/en-us/answers/questions/900512/unable-to-find-replicating-directory-changes-permi) and the reporter, who was running Windows Server 2012 was happy with the response.

I'm not familiar with Windows Server >= 2003, but I don't think much has changed there and see no need to change this. Your new link is definitely not correct, it only is about finding the AD schema, not about granting (replication) privileges.

If you really want to switch to a method with PowerShell, have a look https://sharepoint.stackexchange.com/questions/105119/grant-replicate-changes-permissions-in-active-directory-with-powershell However I'd rather not link some random Stack Exchange answer if we have not tested it. Unless you know what you are doing, let's keep reffering to the official MS documentation.

**Comment by Amrita42** [comment]
Yes, you are right . I can see the link works as expected and there is nothing to fix here imo hence closing this

**Pull request closed without merging**